adapters/onebot/src/bot/message.ts

```diff
@@ -1,9 +1,10 @@
 import { Messenger, pick, segment, Universal } from '@satorijs/satori'
 import { BaseBot } from './base'
 import { CQCode } from './cqcode'
+import { Author } from '../types'
 
 class State {
-  author: Partial<Universal.Author> = {}
+  data: Partial<Author> | string = {}
```

Comment:
I think you could maintain the original name `author` here. As `State` here are for saving miscellaneous pieces of information.

Reply:
But if `author` is used as the `messageId`, it doesn't quite fit its meaning. Also, the name used by `gocq` is also `data`, so I have named it `data` as well.

Reply:
What do you mean by `used as the messageId`?
As my understanding, the `author` here is referring to the message element `author`, so everything you extracted from the element should be put in it. `data` is instead too broad here.

Reply:
In fact, you could just save the messageId in `author` just as you did for other parameters.

Reply:
well okey